src/main/java/com/playmonumenta/networkrelay/RemotePlayerManagerBungee.java

```diff
@@ -140,11 +144,18 @@ Map<String, JsonObject> callGatherPluginDataEvent(RemotePlayerAbstraction player
 	}
 
 	@Override
-	boolean checkAndRefreshIfLocalPlayer(RemotePlayerAbstraction player) {
+	boolean checkIfLocalPlayer(RemotePlayerAbstraction player) {
 		if (!player.getServerType().equals(RemotePlayerProxy.SERVER_TYPE)) {
 			return false;
 		}
-		return refreshLocalPlayer(player.mUuid);
+		@Nullable ProxiedPlayer localPlayer = ProxyServer.getInstance().getPlayer(player.mUuid);
+		return localPlayer != null && localPlayer.isConnected();
+	}
+
+	@Override
+	void refreshLocalPlayerWithDelay(UUID uuid) {
+		// TODO: was lazy and didn't add a delay for the proxy since we can't switch proxies until transfer packets
```

Comment:
Technically you can connect with one Minecraft instance, then start a new one and connect again without disconnecting. I forget if this kicks your first instance or prevents you from joining on the new one, but either way it's a non-issue.